src/job.ts

```diff
@@ -12,6 +12,12 @@ export interface IJob {
     // ios
     buildType?: IOS.BUILD_TYPES;
     bundleIdentifier?: string;
+    upload?: boolean;
+    options?: {
```

Comment:
Please name the key more descriptive, so it indicates that these are Apple Credentials.